boards/common/nrf52/Makefile.dep

```diff
@@ -2,8 +2,4 @@ ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_nrf_temperature
 endif
 
-ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
-  ifeq (,$(filter nordic_softdevice_ble nrfmin nrf802154,$(USEMODULE) $(USEPKG)))
-    USEMODULE += nimble_netif
-  endif
-endif
+include $(RIOTCPU)/nrf52/Makefile.dep
```

Comment:
Done automatically as `CPU` is defined.

Reply:
https://github.com/RIOT-OS/RIOT/blob/a7cb0a4b7793d40526ff20f79bdbe6a71e138b96/Makefile.dep#L8-L13